build

```diff
@@ -9,6 +9,10 @@ if [[ ! -z "$FAILPOINTS" ]]; then
 	GIT_SHA="$GIT_SHA"-FAILPOINTS
 fi
 
+# Uncomment following if debug under macOS with go 1.11 above, for gdb under 
```

Comment:
@taozhijiang can we detect the OS in bash to do this automatically?

Reply:
Of course we can, but I don't know whether gdb ports under macOS will support compress DWART format in the future.
So we count on most people will take a look on this build script, especially when they meet debug problems.

Reply:
Of course we can, but I don't know whether gdb ports under macOS will support compress DWART format in the future.
So we count on most people will take a look on this build script, especially when they meet debug problems.

Reply:
> @taozhijiang can we detect the OS in bash to do this automatically?

Of course we can, but I don't know whether gdb ports under macOS will support compress DWART format in the future.
So we count on most people will take a look on this build script, especially when they meet debug problems.

Reply:
@taozhijiang 

We can print out a line or something to indicate that. 

As a normal users on mac (i believe very few people will run etcd on mac osx for production), he/she probably just wants to try out etcd... so they do not really care about how we build it, as long as it builds.

Reply:
@xiang90 

As you said, comment is a good hint.
Currently build script print out nothing, follow this rule.